modules/installing-trustyai-service-using-cli.adoc

```diff
@@ -75,11 +75,13 @@ spec:
 ----
 +
 <1> The name of the TrustyAI service instance.
-<2> The storage format for the data, either `DATABASE` or `PVC` (persistent volume claim).
+<2> The storage format for the data, either `DATABASE` or `PVC` (persistent volume claim). We recommend using a database setup for better scalability, performance, and data management in TrustyAI.
```

Comment:
{org-name} recommends that you use a database setup for better scalability, performance, and data management in TrustyAI.